### PR TITLE
Readme list workshops

### PR DESCRIPTION
**Pull request by will-moore**
Update main README to better present links to slides and walkthroughs for recent workshops.

Staged at https://github.com/will-moore/training-repos/tree/README_list_workshops

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,59 @@
...
+ * Workshop intro [slides](https://downloads.openmicroscopy.org/presentations/2018/TOPIM-Crete/OMERO-Friday/)
+ * Workshop [walkthrough pdf](https://downloads.openmicroscopy.org/presentations/2018/TOPIM-Crete/TOPIM_walkthrough.pdf)
+
+2018-06-07 Elmi, University College Dublin, Ireland
+---------------------------------------------------
+A 1-hour workshop on OMERO including OMERO.iviewer, OMERO.figure and OMERO.parade.
+ * Workshop intro [slides](https://downloads.openmicroscopy.org/presentations/2018/ELMI-Dublin/OMERO-Workshop/)
+ * Workshop [walkthrough pdf](https://downloads.openmicroscopy.org/presentations/2018/ELMI-Dublin/elmi_walkthrough.pdf)
+
+2018-05-31 University of Dundee U.K.
```
Comment (mtbc) [inline README.md]: comma after "Dundee"

File: README.md [hunk 2]
```diff
@@ -1,14 +1,59 @@
...
+# Recent Workshops: Presentations and walkthrough documents
+
+2018-07-05/06 TOPIM-tech, Chania, Crete
+---------------------------------------
+A full-day workshop on OMERO including Fiji, Orbit, CellProfiler, OMERO.parade and OMERO.figure.
+ * About the workshop [slides](https://downloads.openmicroscopy.org/presentations/2018/TOPIM-Crete/OMERO-Thursday/)
+ * Workshop intro [slides](https://downloads.openmicroscopy.org/presentations/2018/TOPIM-Crete/OMERO-Friday/)
+ * Workshop [walkthrough pdf](https://downloads.openmicroscopy.org/presentations/2018/TOPIM-Crete/TOPIM_walkthrough.pdf)
+
+2018-06-07 Elmi, University College Dublin, Ireland
```
Comment (mtbc) [inline README.md]: ELMI

**Comment by jburel** [comment]
length should probably be a bullet point so it is more obvious
With the README becoming the list of workshops we need to move out the description of the other components to another file and add a link pointing to the file in the README

**Comment by joshmoore** [comment]
A minor comment: if we're hosting the results under downloads, it might be worth considering if the landing page should be there anyway to simplify the process of keeping it in sync.

**Comment by jburel** [comment]
Adding a landing page under download is a possibility 
This will require resurrecting some old index.html page, adjust the deployment of the presentations (some are added via build some manually),  major work on styling it (in general download page required some tlc).
We moved away from those landing pages under download e.g release, apps page only for legacy

I am not convinced that's a path we want to follow

**Comment by will-moore** [comment]
@jburel Made some more fixes.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,60 @@
-# repos
-A super-module of all the repositories used for training sessions
+# Training Repos
 
-## Data
-The files used during a given session can be found in [data.md](data.md).
+This is the main github repository for our training materials.
```
Comment (jburel) [inline README.md]: GitHub

File: README.md [hunk 4]
```diff
@@ -1,14 +1,60 @@
...
+---------------------------------------
+A longer workshop on OMERO including Fiji, Orbit, CellProfiler, OMERO.parade and OMERO.figure.
+ * **Full-day** workshop
+ * [Presentation](https://downloads.openmicroscopy.org/presentations/2018/TOPIM-Crete/OMERO-Thursday/) about the workshop
+ * Workshop intro [slides](https://downloads.openmicroscopy.org/presentations/2018/TOPIM-Crete/OMERO-Friday/)
+ * Workshop [walkthrough pdf](https://downloads.openmicroscopy.org/presentations/2018/TOPIM-Crete/TOPIM_walkthrough.pdf)
+
+2018-06-07 ELMI, University College Dublin, Ireland
+---------------------------------------------------
+A brief workshop on OMERO including OMERO.iviewer, OMERO.figure and OMERO.parade.
```
Comment (jburel) [inline README.md]: remove brief

File: README.md [hunk 5]
```diff
@@ -1,14 +1,60 @@
...
+2018-03-12/14 University of Dusseldorf, Germany
+-----------------------------------------------
+A workshop at the [Trends In Microscopy](http://www.tim2018.de/) meeting including OMERO.iviewer and OMERO.figure.
+ * **1-hour** workshop
+ * Intro [slides](https://downloads.openmicroscopy.org/presentations/2018/TIM-Dusseldorf/OMERO-Workshop/)
+ * Workshop [walkthrough](https://downloads.openmicroscopy.org/presentations/2018/TIM-Dusseldorf/TIM_Duesseldorf_walkthrough.pdf)
+
+2018-02-09 James Hutton Institute, U.K.
+---------------------------------------
+Walkthrough for Facility Manager and Restricted Admin features.
```
Comment (jburel) [inline README.md]: 1h30 hour

File: README.md [hunk 6]
```diff
@@ -1,14 +1,60 @@
...
+ * Workshop [walkthrough pdf](https://downloads.openmicroscopy.org/presentations/2018/HuttonInstitute-Invergowrie/Hutton_Institute_walkthrough.pdf)
+
+2018-01-10 University of Dundee, U.K.
+-------------------------------------
+Walkthrough and general introduction about OMERO.
```
Comment (jburel) [inline README.md]: 1h

**Review by jburel — COMMENTED** [review]
Few comments 
We can review the content of the about in a follow up PR

**Comment by jburel** [comment]
I made the suggested changes. If we do not deploy on time the training page on www
we will need to remove the link from the README

**Comment by jburel** [comment]
Thanks @will-moore

**Pull request merged**